### PR TITLE
Add maxOfVector function

### PR DESCRIPTION
**Pull request by ms10335**

### CODE REVIEW

**Comment by ziobron** [comment]
✅ CI Passed - Task: max-of-vector
🏅 Granted 5 XP for @ms10335

**Comment by ziobron** [comment]
⏱ 3 XP extra for @ms10335 for on-time delivery

File: homework/max-of-vector/maxOfVector.hpp [hunk 1]
```diff
@@ -4,5 +4,20 @@
...
-    return {};
+
+    if(vec.size() == 0)
+    {
+        return {};
+    }
+
+    int returnMaxVal = std::numeric_limits<int>::min();
+
+    for(auto it: vec)
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:15]: Add space after while, if, else etc...

File: homework/max-of-vector/maxOfVector.hpp [hunk 2]
```diff
@@ -4,5 +4,20 @@
...
+    {
+        if(it > returnMaxVal)
```
Comment (nauka-programowania-MA) [inline homework/max-of-vector/maxOfVector.hpp:17]: Add space after while, if, else etc...

**Review by nauka-programowania-MA — APPROVED** [review]

**Pull request closed without merging**